opendata_module/opmon_opendata/tests/test_utils.py

```diff
@@ -130,7 +130,7 @@ def __init__(self, mock_cursor):
     def __enter__(self):
         return self
 
-    def cursor(self):
+    def cursor(self, name=None, withhold=None):
```

Comment:
Why these parameters are added? 🤔

Reply:
To fix broken tests. It was broken by new parameters introduced into PG cursor:
https://github.com/nordic-institute/X-Road-Metrics/blob/develop/opendata_module/opmon_opendata/api/postgresql_manager.py#L121

Reply:
Oh, thanx for the clarification. Type hints would be nice then.

Reply:
Done